Bottom: 9866bab7870219b6988dab9ba93f6d25dc2c100c
Top:    ec3605354c18a37252d7488b1d2137114ebe08c6
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2013-08-13 22:08:59 -0300

Thread safety documentation.

for ChangeLog

	* manual/libdl.texi: New.


---

diff --git a/manual/libdl.texi b/manual/libdl.texi
new file mode 100644
index 0000000..e3fe045
--- /dev/null
+++ b/manual/libdl.texi
@@ -0,0 +1,10 @@
+@c FIXME these are undocumented:
+@c dladdr
+@c dladdr1
+@c dlclose
+@c dlerror
+@c dlinfo
+@c dlmopen
+@c dlopen
+@c dlsym
+@c dlvsym
